src/platform/crypt_openssl.c

```diff
@@ -323,22 +515,23 @@ CxPlatHpKeyCreate(
     Key->CipherCtx = EVP_CIPHER_CTX_new();
     if (Key->CipherCtx == NULL) {
         QuicTraceEvent(
-            LibraryError,
-            "[ lib] ERROR, %s.",
-            "Cipherctx alloc failed");
+            AllocFailure,
+            "Allocation of '%s' failed. (%llu bytes)",
+            "EVP_CIPHER_CTX_new",
+            0);
         Status = QUIC_STATUS_OUT_OF_MEMORY;
         goto Exit;
     }
 
     switch (AeadType) {
     case CXPLAT_AEAD_AES_128_GCM:
-        Aead = EVP_aes_128_ecb();
+        Aead = CXPLAT_AES_128_ECB_ALG_HANDLE;
         break;
     case CXPLAT_AEAD_AES_256_GCM:
-        Aead = EVP_aes_256_ecb();
+        Aead = CXPLAT_AES_256_ECB_ALG_HANDLE;
         break;
     case CXPLAT_AEAD_CHACHA20_POLY1305:
-        Aead = EVP_chacha20();
+        Aead = CXPLAT_CHACHA20_POLY1305_ALG_HANDLE;
```

Comment:
@paulidale this is the diff that is causing the test failures. For Header Protection, we use just `chacha20`, not including the `Poly1305`.

Reply:
Thanks, I'm must have been blind to not notice this.